docs/cli.md

```diff
@@ -3,7 +3,30 @@ meta:
   - name: description
     content: "Fairwinds Polaris | Learn your CLI options"
 ---
-#### CLI Options
+
+## Installation
+
+### From GitHub Releases
+Visit the [releases page](https://github.com/FairwindsOps/polaris/releases) to find the release
+that's right for your environment. For example, on Linux:
+```
+curl -L "https://github.com/FairwindsOps/polaris/releases/download/4.2.0/polaris_linux_amd64.tar.gz" > polaris.tar.gz
+tar -xvf polaris.tar.gz
```

Comment:
you do not need to `chmod +x polaris`?

Reply:
Nope, the tarball has the right permissions inside it.